test/JDBC/expected/BABEL-UNSUPPORTED.out

```diff
@@ -527,6 +527,12 @@ GO
 
 ~~ERROR (Message: only "sql_latin1_general_cp1_ci_as" supported for default collation)~~
 
+DROP DATABASE db_unsupported9;
+GO
+~~ERROR (Code: 911)~~
+
+~~ERROR (Message: database "db_unsupported9" does not exist)~~
```

Comment:
Why do we need this?

Reply:
In this test, db_unsupported9 is being created and is not dropped, this is causing interference with other test files when server collation name is `Arabic_CI_AS`